src/command-reader.ts

```diff
@@ -24,28 +24,24 @@ class CommandReader {
             const commandName = command as HiddenCommandName;
             const commandInfo: HiddenCommand = hiddenCommands[commandName];
             // Only maintainers can use privileged message-based commands
-            if (commandInfo.privileged && !state.isMaintainer(msg.author.id)) {
-                msg.channel.send('You can\'t do that');
-            } else if (commandInfo.failIfDisabled && state.isDisabled()) {
-                msg.channel.send('I can\'t do that while I\'m disabled');
-            } else {
-                try {
-                    if (typeof commandInfo.fn === 'function') {
-                        commandInfo.fn(msg, rawArgs, ...args);
-                        logger.log(`Executed command '${command}' with args ${JSON.stringify(args)}`, MultiLoggerLevel.Warn);
-                    } else {
-                        msg.channel.send(`Warning: deprecated command format does not exist for this command: ${command}`);
-                    }
-                } catch (err) {
-                    if (err instanceof Error) {
-                        logger.log(`Uncaught error while trying to execute command '${msg.content}': ${err.toString()}`, MultiLoggerLevel.Error);
+            if (state.isMaintainer(msg.author.id)) {
```

Comment:
Tbh let's just stop command-reader from even being invoked in the first place (in `bot.js` ignore the message altogether). Thoughts?

Reply:
Don't you still want access to some of these commands?

Reply:
Oh I see, just move this condition into the messageCreate handler